src/common/components/select-repositories-page/select-repositories-page.css

```diff
@@ -16,6 +16,7 @@
 .selectRepositoriesBox {
   border: 1px solid $mid-grey;
   border-radius: 2px;
+  position: relative;
```

Comment:
Is this needed for anything now? I guess it's just a leftover from previous absolute positioning.